app/Controller/ShadowAttributesController.php

```diff
@@ -697,7 +697,7 @@ public function edit($id = null) {
 					foreach ($this->ShadowAttribute->validationErrors as $k => $v) {
 						$message .= '[' . $k . ']: ' . $v[0] . PHP_EOL;
 					}
-					throw new NotFoundException('Could not save the proposal. Errors: ' . $message);
+					throw new NotFoundException(__('Could not save the proposal. Errors: ', $message));
```

Comment:
This is incorrect, missing the %s after "Errors: "

Reply:
f'in Typo ;)
Thanks.